julia_src/http.jl

```diff
@@ -64,50 +64,76 @@ function reopt(req::HTTP.Request)
 			)
 		)
 	end
-    @info "Starting REopt..."
+	@info "Starting REopt..."
     error_response = Dict()
     results = Dict()
 	inputs_with_defaults_set_in_julia = Dict()
-    try
+	model_inputs = nothing
+	# Catch handled/unhandled exceptions in data pre-processing, JuMP setup
+	try
 		model_inputs = reoptjl.REoptInputs(d)
-        results = reoptjl.run_reopt(ms, model_inputs)
-		inputs_with_defaults_from_easiur = [
-			:NOx_grid_cost_per_tonne, :SO2_grid_cost_per_tonne, :PM25_grid_cost_per_tonne, 
-			:NOx_onsite_fuelburn_cost_per_tonne, :SO2_onsite_fuelburn_cost_per_tonne, :PM25_onsite_fuelburn_cost_per_tonne,
-			:NOx_cost_escalation_rate_fraction, :SO2_cost_escalation_rate_fraction, :PM25_cost_escalation_rate_fraction
-		]
-		inputs_with_defaults_from_avert = [
-			:emissions_factor_series_lb_CO2_per_kwh, :emissions_factor_series_lb_NOx_per_kwh,
-			:emissions_factor_series_lb_SO2_per_kwh, :emissions_factor_series_lb_PM25_per_kwh
-		]
-		inputs_with_defaults_set_in_julia = Dict(
-			"Financial" => Dict(key=>getfield(model_inputs.s.financial, key) for key in inputs_with_defaults_from_easiur),
-			"ElectricUtility" => Dict(key=>getfield(model_inputs.s.electric_utility, key) for key in inputs_with_defaults_from_avert)
-		)
-    catch e
-        @error "Something went wrong in the Julia code!" exception=(e, catch_backtrace())
+	catch e
+		@error "Something went wrong in the Julia code!" exception=(e, catch_backtrace())
         error_response["error"] = sprint(showerror, e)
-    end
+	end
+	
+	if isa(model_inputs, Dict) && model_inputs["status"] == "error"
+		results = model_inputs
```

Comment:
There are now two differently structured return dicts depending on the type of error. 1) error_response with key "error"; 2) response with key "results", which has key "status". Can we make them consistent? Will make the code easier to maintain and build on. I think your way is more consistent with a non-error response, so probably better to change the existing error handling to match yours.

Reply:
We could alter `error_response` dictionary to follow formatting of `response` but i would vote to leave it as is. Looks like `error_response` is just used in case of HTTP errors and its involvement is pretty limited in `run_jump_model.py` as well (only line 94).

Reply:
Agreed, it's fine for now. I created an issue.